scripts/release-test-script-ut-core.sh

```diff
@@ -0,0 +1,302 @@
+#!/bin/bash
```

Comment:
no (c) message and these are the requirements ontop of what your doing today, please upgrade the docs to include this.

## ut-core build testing script

|#| TARGET | Docker | Expectation |
|-|--------|--------|-------------|
|1| build TARGET=linux| None|build ut test|
|3| build TARGET=arm| rdk-dunfell| build ut test|
|4| build TARGET=arm| rdk-kirkstone| build ut test|
|5| build TARGET=arm| comcast-vm| build ut test |